src/main/java/zd/zero/waifu/motivator/plugin/listeners/WaifuUnitTesterImpl.java

```diff
@@ -41,6 +42,10 @@ public void stop() {
         this.busConnection.disconnect();
     }
 
+    private boolean isSuccessWithIgnoredTests( SMTestProxy.SMRootTestProxy testsRoot ) {
+        return TestStateInfo.Magnitude.IGNORED_INDEX.equals(  testsRoot.getMagnitudeInfo() );
+    }
+
```

Comment:
Noice! A very simple way to resolve the issue 👌

Reply:
Yeah! I was happy that the fix was straightforward and involved no hacking :)